README.md

```diff
@@ -1,306 +1,265 @@
-# [INFO] [1636706176.184055177] [fake_server_node]: registered a new robot: [fake_ros1_robot]
-# [INFO] [1636706176.184055177] [fake_server_node]: registered a new robot: [fake_ros2_robot]
+cd PATH_TO_EXTRACTED_ZENOH_BRIDGE
+./zenoh-bridge-ros2dds -c ~/ff_ws/src/free_fleet/free_fleet_examples/zenoh_configs/turtlebot3_1_zenoh_config.json5
 ```
 
-ROS 2 messages over the `/fleet_states` topic can also be used to verify that the clients are registered,
+Listen to transforms over `zenoh`,
 
 ```bash
-source ~/ff_ros2_ws/install/setup.bash
-ros2 topic echo /fleet_states
+source ~/ff_ws/install/setup.bash
+ros2 run free_fleet_examples test_tf.py \
+    --namespace turtlebot3_1
 ```
 
-Next, to send requests and commands, check out the example scripts and their uses [here](#commands-and-requests).
-
-</br>
-
-### Turtlebot3 Fleet Server
-
-This launches a server for a fleet of simulated turtlebot3 robots.
+Start a `navigate_to_pose` action over `zenoh`, using example values,
 
 ```bash
-source ~/ff_ros2_ws/install/setup.bash
-ros2 launch ff_examples_ros2 turtlebot3_world_ff_server.launch.xml
+source ~/ff_ws/install/setup.bash
+ros2 run free_fleet_examples test_navigate_to_pose.py \
+    --frame-id map \
+    --namespace turtlebot3_1 \
+    -x 1.808
+    -y 0.503
 ```
 
-At this point, the server should register any clients running, if any of the simulations below are running.
-
-Users can issue requests and commands through the server to the clients. Check out the example scripts and their uses [here](#commands-and-requests).
-
-### ROS 1 Turtlebot3 Simulation
-
-Before starting these examples, remember to install all the prerequisites according to the [official tutorials](https://emanual.robotis.com/docs/en/platform/turtlebot3/quick-start/#pc-setup) of using `Turtlebot3`, under `noetic`.
+Start the RMF core packages on a different `ROS_DOMAIN_ID` to simulate running on a different machine,
 
 ```bash
-sudo apt install ros-noetic-dwa-local-planner
+source ~/ff_ws/install/setup.bash
+export ROS_DOMAIN_ID=55
+
+ros2 launch free_fleet_examples turtlebot3_world_rmf_common.launch.xml
 ```
 
-Launch the basic simulation of a single Turtlebot3, with a free fleet client attached to it, by sourcing the ROS 1 workspace and launching the provided example launch file,
+Launch the `free_fleet_adapter` with the current example's configurations, verify that `turtlebot3_1` has been added to fleet `turtletbot`.
 
 ```bash
-source ~/ff_ros1_ws/install/setup.bash
-export TURTLEBOT3_MODEL=burger; roslaunch ff_examples_ros1 turtlebot3_world_ff.launch
-```
+source ~/ff_ws/install/setup.bash
+export ROS_DOMAIN_ID=55
 
-This launch file starts the simulation in `gazebo`, visualization in `rviz`, as well as the simulated navigation stack of the single turtlebot3. Once the simulation and visualization show up, the robot can be commanded as per normal through `rviz` with `2D Nav Goal`.
+ros2 launch free_fleet_examples tb3_simulation_fleet_adapter.launch.xml
```

Comment:
Hmmm, I definitely see the merit in that, however I was hoping to get users more familiar with the architecture of RMF, rather than launching and killing everything in one go, with this setup, they can jumble it together if they want to, or they can kill just the fleet adapter, fix something, and launch it again, without killing the entire stack

I'll keep this in mind and monitor the UX for a bit, but I'd probably like to maintain these set of steps for now